docs/how-to-configure-new-repository.md

```diff
@@ -16,59 +16,44 @@ for the repository.
 
 1. Every repository has three teams associated with it. Typically for the
    repository `opentelemetry-foo` they will be named `foo-triagers`, `foo-approvers`,
-   and `foo-maintainers`. `foo-maintainers` is a child of `foo-approvers`, and 
-   `foo-approvers` is a child of `foo-triagers`, as it each group always contains 
+   and `foo-maintainers`. `foo-maintainers` is a child of `foo-approvers`, and
```

Comment:
Does "child" mean a subset? Could we say a subset?

Reply:
github calls them [child teams](https://docs.github.com/en/organizations/organizing-members-into-teams/requesting-to-add-a-child-team), though I always have to think twice about which is the parent and which is the child

maybe instead something like?

```suggestion
   and `foo-maintainers`. `foo-maintainers` team is a member of the `foo-approvers` team, and
```

Reply:
I re-wrote this section a bit to make it clearer